chapters/operatorsandexpressions.tex

```diff
@@ -531,10 +531,10 @@ \subsection{Numeric Functions and Conversion Functions}\label{numeric-functions-
   \lstinline!(if leftJustified then "-" else "") + String(minimumLength) + "d"!
 \end{itemize}
 
-Form of the \lstinline!format! string:
-According to ANSI-C the format string specifies one conversion specifier (excluding the leading \%), shall not contain length modifiers, and shall not use `\lstinline!*!' for width and/or precision.
-For all numeric values the format specifiers `\lstinline!f!', `\lstinline!e!', `\lstinline!E!', `\lstinline!g!', `\lstinline!G!' are allowed.
-For integral values it is also allowed to use the `\lstinline!d!', `\lstinline!i!', `\lstinline!o!', `\lstinline!x!', `\lstinline!X!', `\lstinline!u!', and `\lstinline!c!' format specifiers (for non-integral values a tool may round, truncate or use a different format if the integer conversion characters are used).
+The ANSI-C style \lstinline!format! string (which cannot be combined with any of the other named arguments) consists of a single conversion specifier without the leading \%.
```

Comment:
```suggestion
The ANSI-C style \lstinline!format! string (which cannot be combined with any of the other named arguments) consists of a single conversion specification without the leading \%, and it may optionally include flags, minimum field width, and precision specifiers.
```
The terminology is that "%+4.17g" is a conversion specification and "g" is the conversion character.
(Same from K&R and n1570.)

Reply:
The "and it may..." could possibly be dropped.

Reply:
OK, I changed to the original terminology, even though I'm not sure it's the most widely recognized nowadays.  I think we can leave out the _and it may_ to keep the sentence short, and because it is sort of explained in the same paragraph anyway.  Resolving.